src/routes/Project/components/Table.js

```diff
@@ -92,11 +92,26 @@ export default class Table extends Component {
 
     // only left mouse button
     if (e.button !== 0) return;
+
+    // dont allow dragging on ctrl click since this brings up context menu
```

Comment:
ya it does. i honestly dont remember why i did this and my comment is kinda shitty but it was important